docs/release-notes.md

```diff
@@ -14,6 +14,36 @@ toc_float: true
 {: .det}
 <details open markdown="block">
 
+<summary> Version 3.13.0 </summary>
+
+# Version 3.13.0
+Release date: September 12, 2021
+{: .overview}
+
+- Supporting RTL.
+- Context attachment on every request.
+- Upload icon configurable.
+  
+### Fixes
+
+- Read more fix.
+- Fixing issue when the element is not found in the ChatRecorder.
+- Support extra data while starting live chat.
+- Fix defualt read more title size.
+- Minor bug fixes.
```

Comment:
Add - Fixed a bug in which the placeholder hint for the input field was not displayed after the message was auto-corrected